config/settings.yml

```
@@ -28,22 +28,22 @@ scan_pilot_groups:
 origin_admin_groups:
   ART: ['sul:requests-art']
   EDUCATION: ['sul:requests-education']
-  HOPKINS: ['sul:requests-hopkins']
-  HV-ARCHIVE: ['sul:requests-hoover-archive']
-  HOOVER: ['sul:requests-hoover-library']
-  RUMSEYMAP: ['sul:requests-rumsey']
+  MARINE-BIO: ['sul:requests-hopkins']
+  HILA: ['sul:requests-hoover-archive']
+  RUMSEY-MAP: ['sul:requests-rumsey']
   SPEC-COLL: ['sul:requests-spec']
 
 origin_location_admin_groups:
-  PAGE-MP: ['sul:requests-branner']
+  EAR-PAGE-MP: ['sul:requests-branner']
```

```suggestion
  SAL3-PAGE-MP: ['sul:requests-branner']
```

I thought EAR-PAGE-MP was the "prepare items to go to SAL3", not the "items are at SAL3" location. I made up this location code, so you should  verify wit with real-world items.

---

I don't understand what you mean.  There are two PAGE-MP locations: SAL3- and EAR-.  My read here was that since we were asking for an admin group called "requests-branner" we'd want to use EAR-.

---

I've changed all instances to SAL3-PAGE-MP, as I guess that is the correct one?